kubera/authentication/authentication_test.go

```diff
@@ -75,3 +75,89 @@ func TestFetchToken(t *testing.T) {
 		})
 	}
 }
+
+// TestCreateNewUser is to create a new user
+func TestCreateNewUser(t *testing.T) {
```

Comment:
Let us have this in a separate _test.go file e.g. authentication/create_user_test.go